apps/api/src/queries/buildViewModelForResource/viewModels/audio-visual/video.view-model.ts

```diff
@@ -9,7 +9,7 @@ import { CoscradContributor } from '../../../../domain/models/user-management/co
 import { isNullOrUndefined } from '../../../../domain/utilities/validation/is-null-or-undefined';
 import { BaseResourceViewModel } from '../base-resource.view-model';
 
-export class VideoViewModel extends BaseResourceViewModel implements IVideoViewModel {
```

Comment:
That's right. Only the responseInterceptor(queryResponse) should be constrained to this type.